rct229/rules/section6/section6rule5.py

```diff
@@ -0,0 +1,112 @@
+from rct229.rule_engine.rule_base import RuleDefinitionBase
+from rct229.rule_engine.rule_list_indexed_base import RuleDefinitionListIndexedBase
+from rct229.rule_engine.user_baseline_proposed_vals import UserBaselineProposedVals
+from rct229.schema.config import ureg
+from rct229.utils.masks import invert_mask
+
```

Comment:
I believe ```pint_sum```, ```get_avg_zone_height``` ```ZERO```, ```normalize_interior_lighting_schedules```, ```compare_schedules``` function are not imported. Could you verify these?

Reply:
Yes, thank you.